charts/vela-core/templates/defwithtemplate/ingress.yaml

```diff
@@ -0,0 +1,57 @@
+# Code generated by KubeVela templates. DO NOT EDIT.
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  name: ingress
```

Comment:
It would be better if an annotation `definition.oam.dev/description` is set.

Reply:
added